### PR TITLE
added my profile

### PR DESCRIPTION
**Pull request by valeriepadel**
I've added my name to the list!

### CODE REVIEW

File: src/assets/persons.js [hunk 1]
```diff
@@ -17176,5 +17176,22 @@ export const data = {
...
+        github: "https://github.com/valeriepadel"
+      },
+      jobTitle: "Front-End Web Developer",
+      location: {
+        city: "Pasay",
+        state: "NCR",
+        country: "Philippines"
+      }
+    }
+  }
```
Comment (abdus) [inline src/assets/persons.js:17195]: there's an error here

**Review by abdus — CHANGES_REQUESTED** [review]
fix errors

File: src/assets/persons.js (hunk 1 above)
Comment (valeriepadel) [reply src/assets/persons.js:17195]: Thank you @abdus 👍

**Review by valeriepadel — COMMENTED** [review]
add

**Pull request merged**